packages/react/src/drawer/docs/overview.mdx

```diff
@@ -39,13 +39,18 @@ relatedPatterns: ['search-filters']
 
 <DoHeading />
 
+- consider using a side action page first - side action pages support more use cases than side drawers
+- only use side drawers for filters, small overflow form actions and related guidance content if you can't provide these interactions on the parent page
+- only use side drawers for external facing forms when you need to maintain the users context of the parent page 
```

Comment:
```suggestion
- only use side drawers for external facing forms when you need to maintain the users context of the parent page 
```
-only use for external (customer-facing) forms if there is a need to maintain some context of the current page, otherwise link to a separate page